web/app/scripts/app.js

```diff
@@ -130,7 +130,7 @@ angular.module('biggraph').filter('decimal', function() {
 angular.module('biggraph').filter('id', function() {
   return function(x) {
     if (x === undefined) { return x; }
-    return x.toLowerCase().replace(/[ !?,.]/g, '-');
+    return x.toLowerCase().replace(/[ !?,./]/g, '-');
```

Comment:
The custom box IDs ended up with slashes. It kind of works, but it's easier if we don't have slashes.